web/src/App.jsx

```diff
@@ -64,15 +66,15 @@ function App() {
         </SimpleLayout>
       );
 
-    if ((phase === STARTUP && status === BUSY) || phase === undefined || status === undefined) {
```

Comment:
I understand that `phase === undefined` is either, not possible anymore or nor relevant here. Could you please confirm it?

Reply:
Yes, that's it. We are using `suspense: true` to get the installer status.